modules/apps/friendly-url/friendly-url-service/src/main/java/com/liferay/friendly/url/internal/util/FriendlyURLNormalizerImpl.java

```diff
@@ -165,12 +165,19 @@ else if (Arrays.binarySearch(_REPLACE_CHARS, c) >= 0) {
 		return friendlyURL;
 	}
 
+	@Override
+	public String normalizeWithPeriods(String friendlyURL) {
+		return normalize(friendlyURL, true, false);
+	}
+
 	@Override
 	public String normalizeWithPeriodsAndSlashes(String friendlyURL) {
-		return normalize(friendlyURL, true);
+		return normalize(friendlyURL, true, true);
 	}
 
-	protected String normalize(String friendlyURL, boolean periodsAndSlashes) {
+	protected String normalize(
```

Comment:
This can be changed to private now?

Reply:
I will do that. Thanks for the review! ❤️